perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -570,6 +570,11 @@ public void initialize() throws InternalErrorException {
 		}
 	}
 
+	public void destroy() {
+		log.debug("destroying");
+		this.extSourcesManagerBl.destroy();
+	}
```

Comment:
Please add at least small javadoc here with information, that this method is usually called when Perun is going down to clean all opened Resources. I know it is standard behavior, but it is little confusing to see it destroying only extSources manager.

Reply:
done